contracts/test/v0.8/foundry/BaseTest.t.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+
+contract BaseTest is Test {
+    address internal constant OWNER = 0x00007e64E1fB0C487F25dd6D3601ff6aF8d32e4e;
+
+    function setUp() public virtual {
+        // Set the sender to OWNER permanently
```

Comment:
nit: this comment might confuse someone unfamiliar with forge
`// Set the sender to OWNER until changePrank is run again`